charts/nri-kube-events/templates/deployment.yaml

```diff
@@ -75,6 +75,11 @@ spec:
             - name: NRIA_PROXY
               value: "{{ .Values.proxy }}"
             {{- end }}
+            # Using FORWARD_ONLY mode to avoid the entity creation for the events.
+            - name: NRIA_IS_SECURE_FORWARD_ONLY
+              value: "false"
```

Comment:
Is it better to set this to `false`, or empty? Maybe @newrelic/caos can provide some insight.

Reply:
my first approach was to set it to empty but i saw an error log on the agent sayin that was not able to convert it to boolean type, so i followed the `false`.

Reply:
```
level=error msg="unable to interpret environment variables" component=Configuration error="envconfig.Process: assigning NRIA_IS_SECURE_FORWARD_ONLY to IsSecureForwardOnly: converting '' to type bool. details: strconv.ParseBool: parsing \"\": invalid syntax"
```